phpBB/phpbb/template/twig/extension.php

```diff
@@ -182,4 +183,14 @@ function lang()
 
 		return call_user_func_array(array($this->language, 'lang'), $args);
 	}
+
+	/**
+	* Check if a language variable exist
+	*
+	* @return bool
+	*/
+	public function lang_defined($key)
```

Comment:
I'd rather call it `is_lang` but I guess this is fine too. Don't know what others are thinking about this, so just let it that way. Just wanted to share my opinion.

Reply:
I can change if it you want

Reply:
personally I don't really have a preference

Reply:
@rubencm Then just leave it as it is. But change the exist to exists as stated in my previous comment. I'll merge it then.